episodes/main.go

```diff
@@ -0,0 +1,239 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"regexp"
+	"strings"
+
+	"google.golang.org/api/option"
+	"google.golang.org/api/youtube/v3"
+)
+
+type Config struct {
+	TokenFile    string `json:"token_file"`
+	ClientID     string `json:"client_id"`
+	ClientSecret string `json:"client_secret"`
+	PlaylistID   string `json:"playlist_id"`
+}
+
+func getService(ctx context.Context, cfg *Config) (*youtube.Service, error) {
+	tokenSource, err := getTokenSource(ctx, cfg)
+	if err != nil {
+		return nil, err
+	}
+
+	return youtube.NewService(ctx, option.WithTokenSource(tokenSource))
+}
+
+type Episode struct {
+	Dir                string `json:"dir"`
+	ID                 string `json:"id"`
+	Title              string `json:"title"`
+	Description        string `json:"description"`
+	ScheduledStartTime int    `json:"scheduled_start_time"`
+	EnableAutoStart    bool   `json:"enable_auto_start"`
+	EnableAutoStop     bool   `json:"enable_auto_stop"`
+	EnableDvr          bool   `json:"enable_dvr"`
+}
+
+func getNixValue(path string, result any) error {
+	cmd := exec.Command("nix", "eval", "--json", "-f", ".", path)
```

Comment:
```suggestion
	cmd := exec.Command("nix-instantiate", "--eval", "--json", ".", "-A", path)
```

No need to rely on experimental commands for this :)